ginga/AstroImage.py

```diff
@@ -62,7 +61,21 @@ def __init__(self, data_np=None, metadata=None, logger=None,
 
         self.io = ioclass(self.logger)
 
+        # these attributes to be removed--DO NOT USE!!!
+        inherit_primary_header = False
+        if 'inherit_primary_header' in kwargs:
+            warnings.warn("inherit_primary_header kwarg has been deprecated--"
+                          "use save_primary_header when loading image",
+                          PendingDeprecationWarning)
```

Comment:
Not sure if Pending makes sense here. Why not just deprecate for real? Same comment for all similar occurences.

```suggestion
                          DeprecationWarning)
```

Reply:
Hmm, well, I'm ok with that.  I did a little web searching on the correct way to use the deprecation warnings, and it seemed to me that the proscribed wisdom seemed to be that first you should issue a PendingDeprecationWarning to give a heads up that something *will* be deprecated in the release after next, and then (in the next release) change it to a DeprecationWarning, which is sort of a last warning that the feature will be gone in the following release.

Reply:
Problem is I don't think PendingDeprecationWarning even shows up in the logs unless you specifically enable it.

https://docs.python.org/3/library/exceptions.html#PendingDeprecationWarning

Reply:
Changed all `PendingDeprecationWarning` to `DeprecationWarning`